cypress/e2e/propertyAdd/propertyAdd.js

```diff
@@ -74,6 +74,10 @@ And("I choose the option 'Yes' for field 'Is LBH property?'", () => {
     cy.get('[data-testid="is-council-property-yes"]').click()
 })
 
+And("I choose the option 'Yes' for field 'Add Default SOR Contracts'", () => {
```

Comment:
I'd expect to see this in use in a .feature file. Is it not in use in propertyAdd.feature?

Reply:
added